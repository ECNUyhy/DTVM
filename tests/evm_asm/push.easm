PUSH32 15
PUSH2 12
