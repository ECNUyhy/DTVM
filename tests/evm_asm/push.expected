status: success
error_code: 0
stack:
  - '0C'
  - '0F'
memory: ''
storage: {}
transient_storage: {}
return: ''
events: []
